routes/common/postgres_helper.rb

```diff
@@ -253,4 +255,14 @@ def failover
 
     Serializers::Postgres.serialize(@resource, {detailed: true})
   end
+
+  def send_notification_mail_to_partners(resource, user_email)
+    if resource.flavor == PostgresResource::Flavor::PARADEDB
+      Util.send_email(Config.postgres_paradedb_notification_email, "New ParadeDB Postgres database has been created.",
+        greeting: "Hello ParadeDB team,",
+        body: ["New ParadeDB Postgres database has been created.",
```

Comment:
Should we include additional details such as VM size, storage, and HA, or should we share them in private?

Reply:
Makes sense, added additional details